### PR TITLE
Update MSTest runner extensions info

### PR DESCRIPTION
**Pull request by Evangelink**
## Summary

- List licensing model for each extension
- Fix Code Coverage command line options
- Use released versions

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/testing/unit-testing-mstest-runner-extensions.md](https://github.com/dotnet/docs/blob/963b522df4cae708222562747ffda1ca72ae4244/docs/core/testing/unit-testing-mstest-runner-extensions.md) | [MSTest runner extensions](https://review.learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-mstest-runner-extensions?branch=pr-en-us-39152) |
| [docs/core/testing/unit-testing-mstest-runner-intro.md](https://github.com/dotnet/docs/blob/963b522df4cae708222562747ffda1ca72ae4244/docs/core/testing/unit-testing-mstest-runner-intro.md) | [MSTest runner overview](https://review.learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-mstest-runner-intro?branch=pr-en-us-39152) |
| [docs/core/testing/unit-testing-mstest-runner-runsettings.md](https://github.com/dotnet/docs/blob/963b522df4cae708222562747ffda1ca72ae4244/docs/core/testing/unit-testing-mstest-runner-runsettings.md) | [Use runsettings with MSTest runner](https://review.learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-mstest-runner-runsettings?branch=pr-en-us-39152) |

### CODE REVIEW

File: docs/core/testing/unit-testing-mstest-runner-extensions.md [hunk 1]
```diff
@@ -86,7 +92,9 @@ To determine what proportion of your project's code is being tested by coded tes
 
 ### Microsoft code coverage
 
-Microsoft Code Coverage analysis is possible for both managed (CLR) and unmanaged (native) code. Both static and dynamic instrumentation are supported. This extension is shipped as part of [Microsoft.Testing.Platform.Extensions.CodeCoverage](https://nuget.org/packages/Microsoft.Testing.Platform.Extensions.CodeCoverage) NuGet package.
+Microsoft Code Coverage analysis is possible for both managed (CLR) and unmanaged (native) code. Both static and dynamic instrumentation are supported. This extension is shipped as part of [Microsoft.Testing..Extensions.CodeCoverage](https://nuget.org/packages/Microsoft.Testing.Extensions.CodeCoverage) NuGet package.
```
Comment (Evangelink) [inline docs/core/testing/unit-testing-mstest-runner-extensions.md:95]:
```suggestion
Microsoft Code Coverage analysis is possible for both managed (CLR) and unmanaged (native) code. Both static and dynamic instrumentation are supported. This extension is shipped as part of [Microsoft.Testing.Extensions.CodeCoverage](https://nuget.org/packages/Microsoft.Testing.Extensions.CodeCoverage) NuGet package.
```

File: docs/core/testing/unit-testing-mstest-runner-intro.md [hunk 2]
```diff
@@ -41,16 +41,16 @@ To enable the MSTest runner in a MSTest project, you need to add the `EnableMSTe
...
     <!-- 
       Coverlet collector isn't compatible with MSTest runner, you can 
       either switch to Microsoft CodeCoverage (as shown below),
       or switch to be using coverlet global tool
       https://github.com/coverlet-coverage/coverlet#net-global-tool-guide-suffers-from-possible-known-issue
     --> 
-    <PackageReference Include="Microsoft.Testing.Platform.Extensions.CodeCoverage" 
-                      Version="17.10.0-preview.23622.1" />
+    <PackageReference Include="Microsoft.Testing.Extensions.CodeCoverage" 
+                      Version="17.10.0" />
```
Comment (Evangelink) [inline docs/core/testing/unit-testing-mstest-runner-intro.md:53]:
```suggestion
                      Version="17.10.1" />
```

File: docs/core/testing/unit-testing-mstest-runner-extensions.md [hunk 3]
```diff
@@ -86,7 +92,11 @@ To determine what proportion of your project's code is being tested by coded tes
 
 ### Microsoft code coverage
 
-Microsoft Code Coverage analysis is possible for both managed (CLR) and unmanaged (native) code. Both static and dynamic instrumentation are supported. This extension is shipped as part of [Microsoft.Testing.Platform.Extensions.CodeCoverage](https://nuget.org/packages/Microsoft.Testing.Platform.Extensions.CodeCoverage) NuGet package.
+Microsoft Code Coverage analysis is possible for both managed (CLR) and unmanaged (native) code. Both static and dynamic instrumentation are supported. This extension is shipped as part of [Microsoft.Testing.Extensions.CodeCoverage](https://nuget.org/packages/Microsoft.Testing.Extensions.CodeCoverage) NuGet package. 
+
+Unmanaged (native) code coverage is disabled in the extension by default. Use flags `EnableStaticNativeInstrumentation` and `EnableDynamicNativeInstrumentation` to enable it if needed. For more information about unmanaged code coverage, see [Static and dynamic native instrumentation](/visualstudio/test/customizing-code-coverage-analysis?#static-and-dynamic-native-instrumentation).
```
Comment (Evangelink) [inline docs/core/testing/unit-testing-mstest-runner-extensions.md:97]:
```suggestion
> [!NOTE]
> Unmanaged (native) code coverage is disabled in the extension by default. Use flags `EnableStaticNativeInstrumentation` and `EnableDynamicNativeInstrumentation` to enable it if needed. 
> For more information about unmanaged code coverage, see [Static and dynamic native instrumentation](/visualstudio/test/customizing-code-coverage-analysis?#static-and-dynamic-native-instrumentation).
```

File: docs/core/testing/unit-testing-mstest-runner-extensions.md [hunk 4]
```diff
@@ -86,7 +92,13 @@ To determine what proportion of your project's code is being tested by coded tes
 
 ### Microsoft code coverage
 
-Microsoft Code Coverage analysis is possible for both managed (CLR) and unmanaged (native) code. Both static and dynamic instrumentation are supported. This extension is shipped as part of [Microsoft.Testing.Platform.Extensions.CodeCoverage](https://nuget.org/packages/Microsoft.Testing.Platform.Extensions.CodeCoverage) NuGet package.
+Microsoft Code Coverage analysis is possible for both managed (CLR) and unmanaged (native) code. Both static and dynamic instrumentation are supported. This extension is shipped as part of [Microsoft.Testing.Extensions.CodeCoverage](https://nuget.org/packages/Microsoft.Testing.Extensions.CodeCoverage) NuGet package.
```
Comment (Evangelink) [inline docs/core/testing/unit-testing-mstest-runner-extensions.md:95]:
```suggestion
Microsoft Code Coverage analysis is possible for both managed (CLR) and unmanaged (native) code. Both static and dynamic instrumentation are supported. This extension is shipped as part of [Microsoft.Testing.Extensions.CodeCoverage](https://nuget.org/packages/Microsoft.Testing.Extensions.CodeCoverage) NuGet package.
```

File: docs/core/testing/unit-testing-mstest-runner-extensions.md [hunk 5]
```diff
@@ -86,7 +92,13 @@ To determine what proportion of your project's code is being tested by coded tes
 
 ### Microsoft code coverage
 
-Microsoft Code Coverage analysis is possible for both managed (CLR) and unmanaged (native) code. Both static and dynamic instrumentation are supported. This extension is shipped as part of [Microsoft.Testing.Platform.Extensions.CodeCoverage](https://nuget.org/packages/Microsoft.Testing.Platform.Extensions.CodeCoverage) NuGet package.
+Microsoft Code Coverage analysis is possible for both managed (CLR) and unmanaged (native) code. Both static and dynamic instrumentation are supported. This extension is shipped as part of [Microsoft.Testing.Extensions.CodeCoverage](https://nuget.org/packages/Microsoft.Testing.Extensions.CodeCoverage) NuGet package. 
+
+> [!NOTE]
+> Unmanaged (native) code coverage is disabled in the extension by default. Use flags `EnableStaticNativeInstrumentation` and `EnableDynamicNativeInstrumentation` to enable it if needed.
```
Comment (Evangelink) [inline docs/core/testing/unit-testing-mstest-runner-extensions.md:98]:
```suggestion
> Unmanaged (native) code coverage is disabled in the extension by default. Use flags `EnableStaticNativeInstrumentation` and `EnableDynamicNativeInstrumentation` to enable it if needed.
```

**Comment by Evangelink** [comment]
@IEvangelist could we get this merged soon (we went live yesterday) please?

**Review by IEvangelist — APPROVED** [review]

**Pull request merged**